CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.6.3] - 2023-04-12
+
+### Added
+
+- Entry API responses now include the `created_at` field (#3975)
+- `spire-server agent list` CLI command now shows if agents can be re-attested and supports `by_can_reattest` filtering (#3880)
+- `spire-server entry create`, `spire-server entry show` and `spire-server entry update` CLI commands now support hint information, to disambiguate entries with the same SPIFFE ID (#3926, #3787)
+
+### Changed
+
+- X509 CAs minted by an UpstreamAuthority plugin no longer require to have a URI SAN (#3997)
```

Comment:
I would move this under "Fixed" since it was a regression.

```suggestion
- Regression causing X509 CAs minted by an UpstreamAuthority plugin to be rejected if they did not have a URI SAN (#3997)
```